index-4.html

```diff
@@ -1,7 +1,7 @@
 ﻿<!DOCTYPE html>
 <html lang="en">
 <head>
-    <title>Menu</title>
+    <title>Contact</title>
     <meta charset="utf-8">
     <meta name="format-detection" content="telephone=no"/>
     <link rel="icon" href="images/favicon.ico" type="image/x-icon">

```

Comment:
As requested in the project requirement we have to edit the menu to contact, we are not supposed to have a menu and contact.
you can go a head and delete menu